games-action/another_world/another_world-0.0.1.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Another World (NEO-RAW)"
+DESCRIPTION="RAW is a re-implementation of the engine used \
+in the game Another World. This game, released \
+under the name Out Of This World in non-European \
+countries, was written by Eric Chahi at the beginning of the '90s. \
+YOU NEED ORIGINAL GAME FILES (DOS VERSION)."
```

Comment:
I wonder if it wouldn't be a good idea to add the info where to copy the original files here, too. A user may wonder where to put those, and getting easily frustrated, never even starts the game and never sees the alert...

```suggestion
YOU NEED ORIGINAL GAME FILES!
Put the DOS version of 'BANK*' and 'MEMLIST.BIN' in ~/config/non-packaged/data."

Or use finddir and a variable. Or leave it as is, it's just a suggestion. :)
```